rfc/5/index.md

```diff
@@ -711,13 +665,15 @@ The `input` and `output` fields MUST always be included for this transformations
 
 <dl>
   <dt><strong>transformations</strong></dt>
-  <dd>  Each child transformation MUST contain <code>input_axes</code> and <code>output_axes</code> fields
-        whose values are arrays of strings.
-        Every axis name in a child transformation's <code>input_axes</code>
-        MUST correspond to a name of some axis in this parent object's <code>input</code> coordinate system.
-        Every axis name in the parent byDimension's <code>output</code> coordinate system
+  <dd>  MUST be an array of wrapped transformations.
+        Each item MUST contain <code>input_axes</code>, <code>output_axes</code> and <code>transformation</code> fields.
```

Comment:
Wrapping with `<code>` formats the same as back-ticks but its a bit harder to read html than markdown